MarsRover/OutputFormatter.cs

```diff
@@ -53,5 +53,15 @@ private static bool LocationsAreEqual(Location location1, Location location2)
             if (location1.X == location2.X && location1.Y == location2.Y) return true;
             return false;
         }
+
+        private static string FormatRover(Rover rover)
+        {
+            var direction = rover.Direction;
+            if (direction == Direction.North) return MapDisplay.RoverNorth;
+            if (direction == Direction.South) return MapDisplay.RoverSouth;
+            if (direction == Direction.West) return MapDisplay.RoverWest;
+            if (direction == Direction.East) return MapDisplay.RoverEast;
+            return MapDisplay.Rover;

```

Comment:
This is a perfect opportunity for a switch expression i.e. `return direction switch`